src/qt/rpcconsole.cpp

```diff
@@ -206,7 +206,7 @@ void RPCExecutor::request(const QString &command)
     }
 }
 
-RPCConsole::RPCConsole(QWidget *parent) :
+RPCConsole::RPCConsole(QWidget *parent, Qt::WindowFlags f) :
     QDialog(parent),
```

Comment:
I tested with @barton26 help a similar test with https://github.com/iFoggz/Gridcoin-Research/commit/39f10878d16f4522be666c58fcd7d2acd537e6b9 . The issue I see here is `QDialog(parent),` whould be `QDialog(parent, f),` this should make this work or u can just do what the commit shows instead.